### PR TITLE
virsh_event: Update case for BZ#1925384

### PR DESCRIPTION
**Pull request by cliping**
Set cpu mode to host-model to avoid bug 1925384.

Signed-off-by: lcheng <lcheng@redhat.com>

### CODE REVIEW

**Comment by cliping** [comment]
Before fix:
 (1/1) type_specific.io-github-autotest-libvirt.virsh.event.positive_test.virsh_event.test_events.device-removal-failed_event.no_timeout.loop: ERROR: Could not verify DHCP lease: 52:54:00:b0:fa:49 --> 192.168.122.114 (706.74 s)

After fix:
 (1/1) type_specific.io-github-autotest-libvirt.virsh.event.positive_test.qemu_monitor_event.pretty_option.device-removal-failed_event.no_timeout.loop: PASS (59.15 s)
 (1/1) type_specific.io-github-autotest-libvirt.virsh.event.positive_test.virsh_event.test_events.device-removal-failed_event.no_timeout.loop: PASS (57.41 s)

File: libvirt/tests/src/virsh_cmd/domain/virsh_event.py [hunk 1]
```diff
@@ -172,9 +172,16 @@ def prepare_vmxml_mem(vmxml):
             cpu_idx += 2
             numa_dict_list.append(numa_dict)
             numa_dict = {}
+        if vmxml.xmltreefile.find('cpu'):
+            vmxml_cpu = vmxml.cpu
+            logging.debug("Existing cpu configuration in guest xml:\n%s", vmxml_cpu)
+            vmxml_cpu.mode = 'host-model'
+            vmxml_cpu.remove_elem_by_xpath('/model')
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/domain/virsh_event.py:179]: Is this necessary to be removed?

Comment (cliping) [reply libvirt/tests/src/virsh_cmd/domain/virsh_event.py]: Updated, thanks.

**Review by chunfuwen — APPROVED** [review]
LGTM

File: libvirt/tests/src/virsh_cmd/domain/virsh_event.py [hunk 2]
```diff
@@ -172,9 +172,14 @@ def prepare_vmxml_mem(vmxml):
             cpu_idx += 2
             numa_dict_list.append(numa_dict)
             numa_dict = {}
+        if vmxml.xmltreefile.find('cpu'):
+            vmxml_cpu = vmxml.cpu
+            logging.debug("Existing cpu configuration in guest xml:\n%s", vmxml_cpu)
+            vmxml_cpu.mode = 'host-model'
```
Comment (dzhengfy) [inline libvirt/tests/src/virsh_cmd/domain/virsh_event.py:178]: `<numa/>` is not needed in this case?

Comment (cliping) [reply libvirt/tests/src/virsh_cmd/domain/virsh_event.py:178]: line 159 to line 174 is numa settings.

**Review by dzhengfy — APPROVED** [review]
LGTM

**Pull request merged**